src/bot/logic/awareness.ts

```diff
@@ -215,14 +198,61 @@ export class MatchAwarenessImpl implements MatchAwareness {
                 .getPlayers()
                 .filter((p) => p !== playerData.name && !game.areAlliedPlayers(playerData.name, p));
             const enemy = game.getPlayerData(enemyPlayers[0]);
-            this.mainRallyPoint = getPointTowardsOtherPoint(
-                game,
-                playerData.startLocation,
-                enemy.startLocation,
-                10,
-                10,
-                0,
-            );
+            // 使用findPath找到从我方startLocation到enemy.startLocation的Track路径70%位置
```

Comment:
Let's increase RALLY_POINT_UPDATE_INTERVAL_TICKS since we are now running a (potentially) expensive pathfinding every time. Maybe to 300 or so.